### PR TITLE
Checkpoint of demo for OpenTracing.

### PR DESCRIPTION
**Pull request by rlubke**
The instructions differ from master currently as I removed the instructions for deploying to Kubernetes (haven't had cycles to validated it end-to-end).    Tim, would you have cycles to give it a shot on Kubernetes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,64 +77,68 @@ set PATH=%JAVA_HOME%\bin;%PATH%
...
+For Coherence-JPA, use the following pom:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.toplink</groupId>
+  <artifactId>coherence-jpa</artifactId>
+  <version>14.1.1.0-0-0</version>
```
Comment (tmiddlet2666) [inline README.md:111]: This version looks like it has too many zeros, should be 14.1.1-0-0
Also I am not sure why we need this separate pom the pom and jar are available under a newly installed coherence home.

` find . -name 'coherence-jpa*'
./plugins/maven/com/oracle/toplink/coherence-jpa
./plugins/maven/com/oracle/toplink/coherence-jpa/14.1.1/coherence-jpa-14.1.1.pom
./plugins/maven/com/oracle/toplink/coherence-jpa/14.1.1/coherence-jpa-14.1.1.location
./lib/coherence-jpa.jar`

Comment (rlubke) [reply README.md:111]: Weird.  I wonder why it's not present in the dist file I was testing `coherence-java-14.1.1.0.0b77467.zip`.  I unzipped it and ran:  

```
<~/f/c/r/d/r/c/dist>-> find . -print | ag jpa
./coherence/lib/coherence-jpa.jar
<~/f/c/r/d/r/c/dist>->
```

Comment (tmiddlet2666) [reply README.md:111]: I installed via the installer.

Comment (rlubke) [reply README.md:111]: Okay, I'll update the instructions accordingly.

File: README.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence.jar -DpomFile=$COH
 mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-management.jar -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-management/14.1.1/coherence-management.14.1.1.pom
 mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-rest.jar -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-rest/14.1.1/coherence-rest.14.1.1.pom
 mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-http-netty.jar -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-http-netty/14.1.1/coherence-http-netty.14.1.1.pom
-mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-jpa.jar -DpomFile=<coherence-jpa \(see below\)>.pom
+mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-jpa.jar -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-jpa/14.1.1/coherence-jpa.14.1.1.pom
```
Comment (tmiddlet2666) [inline README.md:89]: This should be 
`mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-jpa.jar -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/toplink/coherence-jpa/14.1.1/coherence-jpa-14.1.1.pom`

Comment (rlubke) [reply README.md:89]: Thanks, should be resolved.

**Pull request merged**